### PR TITLE
Add OpenTitan Working Group

### PR DESCRIPTION
**Pull request by bradjc**
### Pull Request Overview

This pull request commits the newly formed OpenTitan Tock  working group charter to the tock/doc directory.

I listed the "adopted" date as a week from now following the standard review period length.

### Testing Strategy

n/a

### TODO or Help Wanted

1. Comments on the charter (from members and others) are welcome.

2. Thoughts and approval/disapproval from the core team.

### Documentation Updated

- [x] Updated the relevant files in `/docs`, or no updates are required.

### Formatting

- [x] Ran `make formatall`.

### CODE REVIEW

**Comment by bradjc** [comment]
> Looks good to me, minor quibble: we really needed to compress Working Groups to `wg`? Feels a bit antithetical to our general explicit > abbreviated nomenclature preferences.

I completely agree. But, squashing words with a shared last and first letter `workinggroup` is always a little weird, and we don't have a precedent for multi word folders in `/doc`...so I went with `wg`. Maybe that meshes with "/doc". I'm happy to change it.

**Comment by ppannuto** [comment]
Yeesh, our multi-word files are a bit of a mess as well:

- BluetoothLEStack.md CodeReview.md CoreTeam.md OutOfTree.md TockBinaryFormat.md <-- 5 "votes" for CamelCase
    - BluetoothLEStack.md <-- CamelCase + acronym is hard to read
- Getting_Started.md Memory_Isolation.md Memory_Layout.md Mutable_References.md Networking_Stack.md <-- 5 "votes" for _'s

Not to mention the support files `process_memory_layout.png` and `tock-stack.png`

I think I propose `Working_Group` for this and a subsequent rename of the rest. I do not feel strongly about this.

**Comment by bradjc** [comment]
@tock/core-wg Hopefully more people can look at this.

File: doc/wg/opentitan.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- Jon Flatley, Google
+- Alistair Francis, Western Digital
+- Garret Kelly, Google
+- Philip Levis, Stanford University/Google
+- Patrick Mooney, Oxide Computer
+
+## Code Purview
+
+The OT working group will be in charge of (responsible for reviewing and
+approving pull requests for) the following code directories:
```
Comment (hudson-ayers) [inline doc/wg/opentitan.md]: In light of the discussion on the call today that the OpenTitan working group should be given merge capabilities, should this be "responsible for reviewing, approving, and merging pull requests for..."

Comment (phil-levis) [reply doc/wg/opentitan.md]: Added some more details.

File: doc/wg/opentitan.md [hunk 2]
```diff
@@ -24,6 +24,28 @@ The goals of the OpenTitan Tock Working Group (OT) are to:
...
+the group: the exact process is not yet determined and may organically
+evolve as the group gains momentum.
+
+The group has a weekly teleconference call. All working group members  
+participate in the call. Other people may be invited to participate to
+help contribute to particular topics or on-going discussions. The
+working group chair decides who beyond the working group members may
+participate in the call.
+
+The working group published detailed notes of its calls. These will be
```
Comment (hudson-ayers) [inline doc/wg/opentitan.md]:
```suggestion
The working group publishes detailed notes of its calls. These will be
```

**Comment by phil-levis** [comment]
I added notes from 2/7.

**Comment by pftbest** [comment]
Why is the file name `2019-02-07.txt` when it is 2020?

**Comment by phil-levis** [comment]
> Why is the file name `2019-02-07.txt` when it is 2020?

Resolved. Sorry!

**Comment by bradjc** [comment]
Merged! We're now official @tock/opentitan-wg! Thanks to everyone who approved.

**Pull request merged**